src/common/filefn.cpp

```diff
@@ -593,6 +593,14 @@ wxRenameFile(const wxString& file1, const wxString& file2, bool overwrite)
         {
             return true;
         }
+        else
+        {
+            wxLogSysError
+            (
+                _("Failed to rename the file '%s' to '%s' by overwriting the already existing destination file"),
+                file1.c_str(), file2.c_str()
```

Comment:
Do we need those two `c_str()` calls here? I thought it has been safe to pass `wxString` directly to **wx** vararg functions since at least v2.9?

Also, by changing the message, we are losing its translations (the new message is also missing the end period)...